includes/wikia/models/DesignSystemCommunityHeaderModel.class.php

```diff
@@ -358,6 +361,15 @@ public function getExploreMenu(): array {
 				],
 			];
 
+			// If store community add link to explore items
+			if ( $this->isFandomStoreCommunity( $wgCityId ) ) {
```

Comment:
Seem my comment on `isFandomStoreCommunity` - I think it will make this code piece more logical.

Reply:
Are you suggesting to remove this and replace with `if (
array_key_exists( $wgCityId, $wgFandomStoreMap ) )`?

Reply:
Yup!

Reply:
Moved this to API controller as well.